app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\AuthRequest;
+use Illuminate\Support\Facades\Auth;
+
+class AuthController extends Controller
+{
+    protected $userService;
+
+    public function login(AuthRequest $request)
+    {
+        $data = [];
+        $data['email'] = $request->email;
+        $data['password'] = $request->password;
```

Comment:
```suggestion
        $data = ['email' => $request->email,
                       'password' => $request->password]; //array creation can be simplified since you have all data you need right here
```